File: benchmarks/loops/bench.go
package main
import "fmt"
func loop_sum() int64 {
    var sum int64 = 0
    var i int64 = 0
    for i < 10000000 {
        sum += i
        i++
    }
    return sum
}
func main() {
    fmt.Println(loop_sum())
}

File: benchmarks/loops/Bench.java
public class Bench {
    public static long loop_sum() {
        long sum = 0;
        long i = 0;
        while (i < 10000000) {
            sum += i;
            i++;
        }
        return sum;
    }
    public static void main(String[] args) {
        System.out.println(loop_sum());
    }
}
